### DISCUSSION TITLE
Error: resource-exhausted: Quota exceeded.

### DISCUSSION THREAD

**Discussion by usman-farooq-dev** [AntlerVC/firetable#910] [Support / Q&A] [answered]
What is meant by this error please explain it. Thanks
![image](https://user-images.githubusercontent.com/66678118/199243936-79ca1f69-1c6c-4674-b3cb-ba5dd9ba4035.png)

**Comment by shamsmosowi** [comment, marked as answer]
when is this happening for you?

**Reply by harinij** [reply]
Hi @Usman-Faooq we have done a bunch of bug fixes and enhancements in the recent releases, if you are still facing this issue please let us know.
